### DISCUSSION TITLE
feed/article/0rLo5erCwbHk8itpwn9q

### DISCUSSION THREAD

**Discussion by giscus** [wisdomekpotu/devhouse#20] [Comments]
# feed/article/0rLo5erCwbHk8itpwn9q

Web site created using create-react-app

https://www.devhouse.live/feed/article/0rLo5erCwbHk8itpwn9q

**Comment by wisdomekpotu** [comment]
i love this article

**Reply by KindEni** [reply]
```javascript
const app = () => 'Hello World'
```
